16.0.0 (#596)
MetaMask/controllers: CHANGELOG.md

## [Unreleased]

## [16.0.0]
### Changed

> - **BREAKING**: Remove Synctokens method from the tokens-service ([#590](https://github.com/MetaMask/controllers/pull/590))

Reviewer: The `tokens-service` is not part of the public API, but the TokenListController is. I've updated the description to mention the controller instead, and to reference the exact name of the method that was removed.
```suggestion
- **BREAKING**: Remove `syncTokens` method from the TokenListController ([#590](https://github.com/MetaMask/controllers/pull/590))
```

> - **BREAKING**: Remove `syncTokens` method from the TokenListController ([#590](https://github.com/MetaMask/controllers/pull/590))

Reviewer: This would make more sense under the `### Removed` category.

>   - This is breaking for any consumers of this method.

Reviewer: Nit: This seems obvious :thinking: We could probably omit this.

> - Enable default caching for token-service ([#594](https://github.com/MetaMask/controllers/pull/594))

Reviewer: Updated to use a more easily understood term for the style of caching we're now using, and to reference the name of the thing being cached rather than the internal API.
```suggestion
- Enable HTTP caching for the dynamic token list managed by the TokenListController ([#594](https://github.com/MetaMask/controllers/pull/594))
```

> - Set tokenList to empty object in test networks ([#588](https://github.com/MetaMask/controllers/pull/588))

Reviewer: This seems to be describing something internal - it doesn't make clear what the actual change was for consumers. It's not clear to me either from reading the PR.

My best guess is that this is a bug fix, and that the change was to clear the dynamic token list when switching to a network not supported by our token list API. Does that sound correct? cc @NiranjanaBinoy @danjm

Reviewer 2: @Gudahtt that is correct

### Fixed

> - Fix bug that allowed for multiple fetch requests instantiate from getGasFeeEstimatesAndStartPolling ([#586](https://github.com/MetaMask/controllers/pull/586))

Reviewer: The wording here seemed a bit confusing to me. Here's a suggested re-wording:
```suggestion
- Fix bug that allowed `getGasFeeEstimatesAndStartPolling` to initiate multiple simultaneous fetch requests ([#586](https://github.com/MetaMask/controllers/pull/586))
```
